en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/S/Sacred_Flame.md

```diff
@@ -8,6 +8,6 @@ _Evocation cantrip_
 **Components:** V, S \
 **Duration:** Instantaneous
 
-As long as you can pinpoint one creature (even if it has cover) radiant holy flame envelops it, dealing 1d8 radiant damage.
+As long as you can pinpoint one creature (even if it has cover) radiant holy flame envelops it, dealing 1d8 radiant damage on a failed Dexterity saving throw.
```

Comment:
Seems I didn't highlight the damage her. So:
```markdown
`1d8`
```
And similarly for the damage rolls in line 13:
```markdown
This spell’s damage increases by `1d8` when you reach 5th level (`2d8`), 11th level (`3d8`), and 17th level (`4d8`).
```